### ISSUE TITLE
🐞 @master/css.webpack - writeVirtualFile is not a function

### ISSUE THREAD

**Issue by itsezc** [master-co/css#107]
### What happened?

Issue with Master CSS Webpack on Next 13 appDir and Webpack

### Version

^2.0.0-beta.86

### Relevant log output

```shell
workshop:next_dev | TypeError: finalInputFileSystem._writeVirtualFile is not a function
workshop:next_dev |     at VirtualModulesPlugin.writeModule (/home/chiru/projects/foretag/node_modules/webpack-virtual-modules/lib/index.js:132:30)
workshop:next_dev |     at i (/home/chiru/projects/foretag/node_modules/@master/css.webpack/dist/index.cjs:1:1063)
workshop:next_dev |     at r.apply (/home/chiru/projects/foretag/node_modules/@master/css.webpack/dist/index.cjs:1:1163)
workshop:next_dev |     at createCompiler (/home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:1434799)
workshop:next_dev |     at /home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:1434423
workshop:next_dev |     at Array.map (<anonymous>)
workshop:next_dev |     at createMultiCompiler (/home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:1434415)
workshop:next_dev |     at create (/home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:1435282)
workshop:next_dev |     at webpack (/home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:1435691)
workshop:next_dev |     at Object.f [as webpack] (/home/chiru/projects/foretag/node_modules/next/dist/compiled/webpack/bundle5.js:28:874364)
workshop:next_dev |     at HotReloader.start (/home/chiru/projects/foretag/node_modules/next/dist/server/dev/hot-reloader.js:466:44)
workshop:next_dev |     at async DevServer.prepare (/home/chiru/projects/foretag/node_modules/next/dist/server/dev/next-dev-server.js:525:9)
workshop:next_dev |     at async /home/chiru/projects/foretag/node_modules/next/dist/cli/next-dev.js:344:17
workshop:next_dev | error: script "dev" exited with code 1 (SIGHUP)
```

### What browsers are you seeing the problem on?

Firefox

**Comment by 1aron** [comment]
Have you tried:
- upgrade the version to 2.0.0-beta.93
- `rm -rf node_modules && rm -rf package-lock.json && npm i`

The Next 13 example is working https://github.com/master-co/css/tree/beta/examples/next.js
Not sure if it's a package manager problem.

<img width="911" alt="Screenshot 2022-12-21 at 1 24 09 PM" src="https://user-images.githubusercontent.com/33840671/208828304-9287956c-8be7-4923-9a0c-c890b0e692ca.png">

By the way, the HMR of Next 13 is not working yet #102 , this is probably a bug of Next.

**Comment by 1aron** [comment]
🎉 This issue has been fixed and released in [v2.0.0-beta.95](https://github.com/master-co/css/releases/tag/v2.0.0-beta.95)
